gobblin-yarn/src/main/java/org/apache/gobblin/yarn/GobblinYarnConfigurationKeys.java

```diff
@@ -124,4 +126,5 @@
   public static final String GOBBLIN_YARN_AZKABAN_CLASS_LOG_LEVELS = GOBBLIN_YARN_PREFIX + "azkaban.class.logLevels";
   //Container classpaths properties
   public static final String GOBBLIN_YARN_ADDITIONAL_CLASSPATHS = GOBBLIN_YARN_PREFIX + "additional.classpaths";
+  public static final String GOBBLIN_YARN_CLASSPATHS = GOBBLIN_YARN_PREFIX + "classpaths";
```

Comment:
Do we need an additional config? Or can we use gobblin.yarn.additional.classpaths?

Reply:
Additional path is to add new class path on the default one which is set the the yarn config. And in our case, we need to overwrite the default config to a different class path, so I just add a new config and keep the old one for backward compatibility and support more flexible use case.